tests/zenko_e2e/backbeat/ReplicationUtility.js

```diff
@@ -561,6 +563,57 @@ class ReplicationUtility {
             return cb();
         });
     }
+    
+    assertNoBlob(containerName, key, cb) {
+        this.azure.getBlobProperties(containerName, key, err => {
+            assert(err.code);
+            return cb();
+        });
+    }
+
+    checkReplicationStatus(locationName, cb) {
+        const hostname = 'zenko.local';
+        let path = '/_/backbeat/api/crr/status';
+        if (locationName) {
+            path = `${path}/${locationName}`;
+        }
+        makeGETRequest(path, (err, res) => {
+            getResponseBody(res, (err, body) => {
```

Comment:
How come no error checking for the use of `makeGETRequest` and `getResponseBody`?

Reply:
fixed